### PR TITLE
chore: added retry for flaky ttranslate test

### LINKED ISSUE #1572: translate/v3: TestBatchTranslateText failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 7d0196c13ba00138822b3c405f5efa5d4ca7a1b1
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/c6128eda-caf7-4735-be1d-01f3c8425156), [Sponge](http://sponge2/c6128eda-caf7-4735-be1d-01f3c8425156)
status: failed
<details><summary>Test output</summary><br><pre>/usr/local/go/src/testing/testing.go:1296 +0xfd
/usr/local/go/src/time/sleep.go:172 +0x44
/usr/local/go/src/testing/testing.go:879 +0x383
/usr/local/go/src/testing/testing.go:1119 +0x78
/usr/local/go/src/testing/testing.go:827 +0xbf
/usr/local/go/src/testing/testing.go:1117 +0x2aa
/usr/local/go/src/testing/testing.go:1034 +0x165
_testmain.go:60 +0x13d
/go/pkg/mod/go.opencensus.io@v0.22.4/stats/view/worker.go:276 +0x100
/go/pkg/mod/go.opencensus.io@v0.22.4/stats/view/worker.go:34 +0x65
/go/pkg/mod/github.com/googleapis/gax-go/v2@v2.0.5/invoke.go:55 +0xd2
/go/pkg/mod/cloud.google.com/go@v0.60.0/longrunning/longrunning.go:156 +0xe6
/go/pkg/mod/cloud.google.com/go@v0.60.0/longrunning/longrunning.go:142 +0xf2
/go/pkg/mod/cloud.google.com/go@v0.60.0/translate/apiv3/translation_client.go:399 +0xaa
/tmpfs/src/github/golang-samples/translate/v3/batch_translate_text.go:73 +0x68d
/tmpfs/src/github/golang-samples/translate/v3/batch_translate_text_test.go:55 +0x54a
/usr/local/go/src/testing/testing.go:827 +0xbf
/usr/local/go/src/testing/testing.go:878 +0x35c
/go/pkg/mod/google.golang.org/grpc@v1.30.0/clientconn.go:1167 +0x64f
/go/pkg/mod/google.golang.org/grpc@v1.30.0/clientconn.go:810 +0xc2
/go/pkg/mod/google.golang.org/grpc@v1.30.0/balancer_conn_wrappers.go:69 +0xbf
/go/pkg/mod/google.golang.org/grpc@v1.30.0/balancer_conn_wrappers.go:60 +0x16d
/usr/local/go/src/runtime/netpoll.go:173 +0x66
/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0x9a
/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x3d
/usr/local/go/src/internal/poll/fd_unix.go:169 +0x179
/usr/local/go/src/net/fd_unix.go:202 +0x4f
/usr/local/go/src/net/net.go:177 +0x68
/usr/local/go/src/crypto/tls/conn.go:492 +0x89
/usr/local/go/src/crypto/tls/conn.go:593 +0xdd
/usr/local/go/src/crypto/tls/conn.go:1145 +0xf1
/usr/local/go/src/bufio/bufio.go:216 +0x22f
/usr/local/go/src/io/io.go:310 +0x88
/usr/local/go/src/io/io.go:329 +0x58
/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:237 +0x7b
/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:492 +0xa3
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:1287 +0x17b
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:309 +0xedf
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/controlbuf.go:395 +0x10e
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/controlbuf.go:513 +0x1ac
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:355 +0x7b
/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:353 +0x109a</pre></details>

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes #1572

### CODE REVIEW

File: translate/v3/batch_translate_text_test.go [hunk 1]
```diff
@@ -51,11 +52,13 @@ func TestBatchTranslateText(t *testing.T) {
...
-	if err := batchTranslateText(&buf, tc.ProjectID, location, inputURI, outputURI, sourceLang, targetLang); err != nil {
-		t.Fatalf("batchTranslateText: %v", err)
-	}
-	if got, want := buf.String(), "Total characters"; !strings.Contains(got, want) {
-		t.Errorf("batchTranslateText got:\n----\n%s----\nWant to contain:\n----\n%s\n----", got, want)
-	}
+	testutil.Retry(t, 3, 10*time.Second, func(r *testutil.R) {
+		var buf bytes.Buffer
+		if err := batchTranslateText(&buf, tc.ProjectID, location, inputURI, outputURI, sourceLang, targetLang); err != nil {
+			t.Fatalf("batchTranslateText: %v", err)
```
Comment (tbpg) [inline translate/v3/batch_translate_text_test.go:58]:
```suggestion
			r.Fatalf("batchTranslateText: %v", err)
			return
```

File: translate/v3/batch_translate_text_test.go [hunk 2]
```diff
@@ -51,11 +52,13 @@ func TestBatchTranslateText(t *testing.T) {
...
+		}
+		if got, want := buf.String(), "Total characters"; !strings.Contains(got, want) {
+			t.Errorf("batchTranslateText got:\n----\n%s----\nWant to contain:\n----\n%s\n----", got, want)
```
Comment (tbpg) [inline translate/v3/batch_translate_text_test.go:61]:
```suggestion
			r.Errorf("batchTranslateText got:\n----\n%s----\nWant to contain:\n----\n%s\n----", got, want)
```

**Review by tbpg — CHANGES_REQUESTED** [review]
Thanks!

**Comment by munkhuushmgl** [comment]
@tbpg Looks like all the checks are passed, PTAL

**Review by tbpg — APPROVED** [review]
Thanks!

**Pull request merged**